src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,213 @@
+                printLine();
+            } catch (IndexOutOfBoundsException exception) {
+                System.out.println(" Stop feeding me things that do not exist!");
+                printLine();
+            } catch (DukeException exception) {
+                System.out.println(exception.toString());
+                printLine();
+            }
+        } while (!command.equals("bye"));
+    }
+    public static void printLine() {
+        String dash = "\u2500";
+        System.out.println(dash.repeat(SIZE_LINE));
+    }
+    public static void printInstructions() {
+        System.out.println(" - Add a todo e.g. todo read book");
+        System.out.println(" - Add a deadline e.g. deadline do quiz /by 1st Sept");
+        System.out.println(" - Add an event e.g. event prom /at Mon 6pm");
+        System.out.println(" - List all added tasks e.g. list");
+        System.out.println(" - Complete a task e.g. done 1");
+        System.out.println(" - Delete a task e.g. delete 1");
+        System.out.println(" - Say bye to me e.g. bye");
+    }
+    public static void printGreeting(){
+        printLine();
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        printInstructions();
+        printLine();
+    }
+    public static void printBye(){
+        System.out.println(" Bye. Hope to see you again soon!");
+    }
+
+    // Decides on the output for each command
+    public static void getCommandOutput(String command) throws DukeException{
+        Scanner taskObj = new Scanner(command);
+        int totalNumOfTasks = Task.getTotalNumOfTasks();
+        // Initialisation of enum
+        typeOfTasks typeOfTask;
+        printLine();
+        switch (taskObj.next()){
+        case "list":
+            printTaskList(totalNumOfTasks);
+            break;
+        case "bye":
+            printBye();
+            break;
+        case "done":
+            markAsDone(taskObj);
+            WriteToFile.updateFile(tasks, Task.getTotalNumOfTasks());
+            break;
+        case "todo":
+            typeOfTask = typeOfTasks.TODO;
+            addTodo(command, typeOfTask);
+            WriteToFile.updateFile(tasks, Task.getTotalNumOfTasks());
+            break;
+        case "deadline":
+            typeOfTask = typeOfTasks.DEADLINE;
+            addDeadline(command, typeOfTask);
+            WriteToFile.updateFile(tasks, Task.getTotalNumOfTasks());
+            break;
+        case "event":
+            typeOfTask = typeOfTasks.EVENT;
+            addEvent(command, typeOfTask);
+            WriteToFile.updateFile(tasks, Task.getTotalNumOfTasks());
+            break;
+        case "delete":
+            removeTask(taskObj);
+            WriteToFile.updateFile(tasks, Task.getTotalNumOfTasks());
+            break;
+        default:
+            System.out.println(" Command's power level too high! Please try something else or improve my power level!");
+            break;
+        }
```

Comment:
It'd be a good idea to refactor this (the processing of the commands) out to another method for greater adherence to SLAP.